libs/base/Data/Strings.idr

```diff
@@ -1,3 +1,4 @@
+||| To be deprecated
 module Data.Strings
 
```

Comment:
Can we not replace the content of this module with `import public Data.String`?

Reply:
This was problematic on my machine. But I seems not able to reproduce it... I'll push it here and let the CI try it.

Reply:
```
Error: While processing right hand side of show. Undefined name Strings.singleton. 

System/Path.idr:110:22--110:44
     |
 110 |   show (Disk disk) = Strings.singleton disk ++ ":"
     |                      ^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
```
Error: While processing right hand side of show. Undefined name Strings.singleton. 

System/Path.idr:110:22--110:44
     |
 110 |   show (Disk disk) = Strings.singleton disk ++ ":"
     |                      ^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
The trick was to import qualified! ;)